src/main/java/com/greenfox/lvlup/config/SetupInitialData.java

```diff
@@ -0,0 +1,128 @@
+package com.greenfox.lvlup.config;
+
+import com.greenfox.lvlup.model.entity.*;
+import com.greenfox.lvlup.repositrory.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+
+@Component
+@Profile(value = "dev")
+public class SetupInitialData implements CommandLineRunner {
+    private BadgeRepository badgeRepository;
+    private BadgeLevelRepository levelRepository;
+    private UserRepository userRepository;
+    private ArchetypeRepository archetypeRepository;
+    private ReviewRepository reviewRepository;
+    private PitchRepository pitchRepository;
+
+    @Value("${spring.datasource.url}")
+    private String dbname;
+
+    @Autowired
+    public SetupInitialData(ArchetypeRepository archetypeRepository, BadgeRepository badgeRepository,
+                            BadgeLevelRepository levelRepository, UserRepository userRepository,
+                            ReviewRepository reviewRepository, PitchRepository pitchRepository) {
+        this.badgeRepository = badgeRepository;
+        this.levelRepository = levelRepository;
+        this.userRepository = userRepository;
+        this.archetypeRepository = archetypeRepository;
+        this.reviewRepository = reviewRepository;
+        this.pitchRepository = pitchRepository;
+    }
+
+    @Override
+    @Transactional
+    public void run(String... args) throws Exception {
+        System.out.println("****************************" + dbname);
+
+
+        Badge badge1 = new Badge("2.2", "Process improver", "general");
```

Comment:
Since we have a huge code here, it would be a good if we could split this big method here. It will help the readability of the code if e.g. we would have 1 method for each repository where we add the seeded data, and we are calling these methods in then specific order in the run method